### ISSUE TITLE
Subtitle control 

### ISSUE THREAD

**Issue by Jamess661** [futo-org/fcast#13]
Will there be a way to add subtitle settings like you have for audio think that would be great the only thing is missing.

**Comment by Michael-Hollister** [comment]
Yes, you should be able to view subtitles in the Fire stick FCast player if you enable them for the video on Grayjay before casting to the Fire stick. However seems like the video does not load after enabling subtitles if you do this, so will look into fixing this bug in a future update.

**Comment by Jamess661** [comment]
I got excited today cuz I just saw there was an update. It looks like on
the fire stick the app uploaded to version 41 from version 40. So I got all
excited thinking that. Oh my God did they fix it already but unfortunately
it's not done. I took that screenshot and I circled so you could see where
you can pick from the different audio formats to play back. But as you can
see, the only thing you can do other than picking the audio is to adjust
the speed of the audio. It would be so great if you could just add an
option right down in the same spot for the subtitles. Okay? Cuz I can't get
subtitles any other way I'm trying to use grayjay to push the subtitles but
having some issues so I just wanted to send it. Sorry this email is so long.

**Comment by Jamess661** [comment]
> Yes, you should be able to view subtitles in the Fire stick FCast player if you enable them for the video on Grayjay before casting to the Fire stick. However seems like the video does not load after enabling subtitles if you do this, so will look into fixing this bug in a future update.

I'm not that familiar with Grayjay, just FYI. I updated an image you can see at the bottom right. The speed and audio toggles... I think one to enable subtitles and/or the options to change between languages would be great because you can already do it with the audio.

**Comment by Jamess661** [comment]
I can change the subtitles now even without greyjay but why was the option
to change the audio tracks removed in this update? would have been great to
have the option to change both.

**Comment by Michael-Hollister** [comment]
Playback settings adjustment like playback speed and audio quality were removed in a previous update when the whole application interface was updated to the current design. Part of the playback settings adjustment options was brought back with this last update (subtitles and speed), and audio tracks/video quality options will likely also be re-added in the next/future update.
